CONTRIBUTING.md

```diff
@@ -122,21 +123,25 @@ The minimum requirements for an android emulator:
 * Minimum API Level 15 (Target API Level is 20)
 * An SD card
 
-This test app can be run via gradle (in the AndroidTest directory of your checkout)
+The tests can be run like any other android application test suite, through
+Android Studio / IntelliJ Idea or via the command line.
 
+You *will* need to run the `uploadFixtures` gradle task before running any tests.
+
+Note: you can run `uploadFixtures` task once per emulator if you don't add any
+test fixtures.
 
 ```bash
-$ ../gradlew clean installStandardTestDebug waitForTestAppToFinishTests
+$ ../gradlew clean uploadFixtures connectedCheck
```

Comment:
Although you mention the directory above, it would be useful to have a list eg
1) Start emulator
2) cd to $directory
3) ../gradlew clean etc etc